industry_benchmarks/utils/data_gathering.py

```diff
@@ -496,6 +540,18 @@ def gather_data(
         fixed_network = parse_ligand_network(fixed_ligand_network)
         ligand_network = ligand_network.enlarge_graph(
             edges=fixed_network.edges, nodes=fixed_network.nodes)
+
+    if hide_ligand_names:
+        ligand_network, name_mapping = replace_ligand_names(ligand_network)
+        with open(output_dir / "ligand_name_mapping.json", "w") as out:
```

Comment:
I forgot, will we in the end zip up this whole output directory and ask them to send it to us or should they send the individual files within the directory? If the former, this file should be stored elsewhere.

Reply:
Ah yeah that's a good point, folks are 100% going to forget to remove the file if it ends up in the same directory.

Maybe it needs to be nested:

output_dir/SENDTHISDIRECTORYTOUS
output_dir/somefilesforyou